src/styles/brackets_patterns_override.less

```diff
@@ -734,11 +734,13 @@ a:focus {
 /* Dialog-related styles */
 
 .modal-wrapper {
-    width: 100%;
+    width: 100% !important;
+    max-width: 75%;
```

Comment:
Why we need max-width ; 75% here as It will not centrally align the popup to the whole Brackets window.